### PR TITLE
Updating Neb mini from Scav.

### PR DESCRIPTION
**Pull request by MistakeNot4892**
- Tons of map changes.
- Removal of the default x4, y4 overmap position for the overmap site.
- Re-themes the map from generic corporate to tradehouse, a la the tradeship.
- Retypes pilot voidsuits to expedition voidsuits for symmetry.

Nobody except Scav uses this map generally and I'm trying to make it easier to do a dev update downstream, so I'm not too fussed about any clobbering.

### CODE REVIEW

**Review by comma — DISMISSED** [review]

**Review by out-of-phaze — DISMISSED** [review]

**Review by out-of-phaze — DISMISSED** [review]

File: code/game/objects/objs.dm [hunk 1]
```diff
@@ -21,7 +21,7 @@
 	var/armor_penetration = 0
 	var/anchor_fall = FALSE
 	var/holographic = 0 //if the obj is a holographic object spawned by the holodeck
-	var/tmp/directional_offset ///JSON list of directions to x,y offsets to be applied to the object depending on its direction EX: @'{"NORTH":{"x":12,"y":5}, "EAST":{"x":10,"y":50}}'
+	var/list/directional_offset ///JSON list of directions to x,y offsets to be applied to the object depending on its direction EX: @'{"NORTH":{"x":12,"y":5}, "EAST":{"x":10,"y":50}}'
```
Comment (MistakeNot4892) [inline code/game/objects/objs.dm:24]: This was needed in order to be able to null the list so switches/buttons mapped onto tables didn't offset.

Comment (out-of-phaze) [reply code/game/objects/objs.dm:24]: Can't you do `var/tmp/list`?

Comment (MistakeNot4892) [reply code/game/objects/objs.dm:24]: `tmp` hides it from the mapper.

Comment (out-of-phaze) [reply code/game/objects/objs.dm:24]: Oh. It doesn't in StrongDMM, so I never noticed. Yeah, fair

**Review by out-of-phaze — APPROVED** [review]

**Pull request merged**